core/src/test/java/com/gentics/mesh/core/node/NodeImageResizeEndpointTest.java

```diff
@@ -223,6 +224,23 @@ public void testResizeWithFocalPoint() throws IOException {
 			call(() -> client().downloadBinaryField(PROJECT_NAME, node.getUuid(), "en", "image", params));
 		}
 	}
+	
+	@Test
+	public void testFocalPointZoomWithTooLargeTarget() throws IOException {
+		try (Tx tx = tx()) {
+			Node node = folder("news");
+			// 1. Upload image
+			NodeResponse response = uploadImage(node, "en", "image");
+			
+			// 2. Zoom into image
+			ImageManipulationParameters params = new ImageManipulationParametersImpl().setWidth(2048).setHeight(2048);
+			params.setFocalPoint(0.5f, 0.5f);
+			params.setFocalPointZoom(1.5f);
+			params.setCropMode(CropMode.FOCALPOINT);
+			call(() -> client().downloadBinaryField(PROJECT_NAME, response.getUuid(), "en", "image", params), BAD_REQUEST,
+					"image_error_target_too_large_for_zoom");
+		}
```

Comment:
Only wrap the tx around code which needs it. Otherwise you might run into issues due to overlapping transactions in different threads.